agrirouter-sdk-dotnet-standard-impl/Service/Messaging/Abstraction/SendMessageBaseService.cs

```diff
@@ -245,8 +247,7 @@ public EncodedMessage Encode(SendChunkedMessageParameters sendMessageParameters)
                 TechnicalMessageType = sendMessageParameters.TechnicalMessageType,
                 Mode = Mode,
                 Recipients = sendMessageParameters.Recipients,
-                ChunkInfo = sendMessageParameters.ChunkInfo,
-                Metadata = sendMessageParameters.Metadata,
+                ChunkInfo = sendMessageParameters.ChunkInfo
```

Comment:
Why was the MetaData-Field removed; Following the  documentation (see https://docs.my-agrirouter.com/agrirouter-interface-documentation/latest/integration/build-message.html ), every chunk shall include the information

Reply:
It has only been added yesterday, as #101 had been merged :) Probably this branch isn't totally up-to-date?